Application/EdFi.Admin.DataAccess/Repositories/ClientAppRepo.cs

```diff
@@ -244,7 +243,6 @@ public async Task<ApiClient> GetClientAsync(string key)
                 return await context.Clients.Include(c => c.Application)
```

Comment:
We also need some tests for this async function. Should probably refactor so that the synchronous function calls the async one, but uses `.Wait` to turn it into a synchronous function. That way we aren't repeating the same logic in both.